app/views/posts/show.html.erb

```diff
@@ -1 +1,20 @@
-<h1>This is the post with id: </h1>
\ No newline at end of file
+<main>
+  <section>
+    <% if @user.last_posts.blank? %>
+      <h2>There are no posts. Please add a post.</h2>
+    <% else %>
+    <div class='final-post'>
+      <h3>Post # <%= @post.id %> by <%= @post.author.name%></h3>
+      <p class='counter'>Comments: <%= @post.comments_counter %>, Likes: <%= @post.likes_counter %></p>
+      <p><%= @post.text %></p>
+
+      <% @post.comments.each do |comment| %>
+        <div class='comments'>
+          <p><%= comment.author.name %>:
+          <p><%= comment.text %>
+        </div>
+      <% end %>
+      </div>
+    <% end %>
+  </section>
+</main>
```

Comment:
- [ ] it would be better to add spacing margin and styles to put the elements as in the wireframe 
- the name of the commented user and the comment should be on the same line 
- the whole content of the page should be centered with margin in the edges 
- so that you match the wireframe design 
- Yours 

<img width="251" alt="Screenshot 2023-06-02 at 11 44 04" src="https://github.com/lugard1/blogapp/assets/61790587/3d76afff-e617-4f99-a37d-2a1a1d4057bd">


- Design 
<img width="452" alt="Screenshot 2023-06-02 at 11 27 09" src="https://github.com/lugard1/blogapp/assets/61790587/da73526c-cc49-40fe-aa3b-6556e68e0888">